dataset/dictionary/characters-khaenriah.ts

```diff
@@ -56,9 +56,20 @@ export default [
   },
   {
     en: "Irmin",
-    ja: "エルミン",
+    ja: "エルミン / イルミン",  //Note: 表記ゆれ
     zhCN: "伊尔明",
     zhTW: "伊爾明",
+    notes: "カーンルイア黒日王朝時代の君主「黒王」",
```

Comment:
Let's write about the inconsistency in the note.

```suggestion
    notes: "カーンルイア黒日王朝時代の君主「黒王」。「エルミン」「イルミン」の表記ゆれあり。",
```